The above OL personal build result:

https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=a7660c44-437c-4738-b5da-38f907105ac9&tab=Tests


<img width="1518" height="338" alt="image" src="https://github.com/user-attachments/assets/97252cca-369d-4293-9b1c-0580c37123ec" />

```
junit.framework.AssertionFailedError: 2025-07-16-03:10:03:037 No TCK test JUnit result files were found in the results directory (/home/ci/Build/workspace/ebcTestRunner/dev/autoFVT/io.openliberty.microprofile.rest.client.3.0.internal_fat_tck/autoFVT/results/tck_tck-suite_EE9_FEATURES_MicroProfile_50/surefire-reports) which suggests the TCK tests did not run
Errors found in mvnOutput were:
### maven test output:
03:10:02,896 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project tck.runner.tck: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
03:10:02,896 [ERROR] Command was /bin/sh -c cd /home/ci/Build/workspace/ebcTestRunner/dev/autoFVT/io.openliberty.microprofile.rest.client.3.0.internal_fat_tck/autoFVT/publish/tckRunner/tck && /home/ci/Build/workspace/ebcTestRunner/dev/jvm/undertest/jdk-21.
```

`testRestClient30Tck` from `io.openliberty.microprofile.rest.client.3.0.internal_fat_tck` failed, where output.txt indicated javacore:

```
[07/16/2025 03:09:46:460 PDT] 037 StreamMonitor                  process-output                 I Jul 16, 2025 3:09:46 AM com.ibm.ws.fat.util.tck.AbstractArchiveWeaver process
[07/16/2025 03:09:46:460 PDT] 037 StreamMonitor                  process-output                 I INFO: com.ibm.ws.fat.util.tck.JettyArchivesProcessor, WLP: Adding Jar: /home/ci/Build/workspace/ebcTestRunner/dev/autoFVT/image/output/wlp/usr/servers/FATServer/jetty-http-9.4.31.v20200723.jar to SubResourceTest.war
[07/16/2025 03:09:46:460 PDT] 037 StreamMonitor                  process-output                 I Jul 16, 2025 3:09:46 AM com.ibm.ws.fat.util.tck.AbstractArchiveWeaver process
[07/16/2025 03:09:46:460 PDT] 037 StreamMonitor                  process-output                 I INFO: com.ibm.ws.fat.util.tck.WiremockArchiveProcessor, Trying to weave into SubResourceTest.war: 41 assets but had nothing to add
[07/16/2025 03:09:50:287 PDT] 037 StreamMonitor                  process-output                 I Unhandled exception
[07/16/2025 03:09:50:287 PDT] 037 StreamMonitor                  process-output                 I Type=Segmentation error vmState=0x00000000
[07/16/2025 03:09:50:287 PDT] 037 StreamMonitor                  process-output                 I J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
[07/16/2025 03:09:50:287 PDT] 037 StreamMonitor                  process-output                 I Handler1=00007FE6B8656340 Handler2=00007FE6B85AE7D0 InaccessibleAddress=00007FE66BAF5305
[07/16/2025 03:09:50:287 PDT] 037 StreamMonitor                  process-output                 I RDI=00007FE66BAF5305 RSI=00007FE6B8C599AE RAX=00007FE6B8BD95C0 RBX=00007FE66BAF5305
[07/16/2025 03:09:50:287 PDT] 037 StreamMonitor                  process-output                 I RCX=00007FE6B8C599AE RDX=000000000000006C R8=00007FE6B4055480 R9=0000000000001998
[07/16/2025 03:09:50:290 PDT] 037 StreamMonitor                  process-output                 I R10=00007FE6B4000560 R11=00007FE6B4000090 R12=0000000000000000 R13=00007FE6B8C599AE
[07/16/2025 03:09:50:290 PDT] 037 StreamMonitor                  process-output                 I R14=00007FE6B50BEFF8 R15=00007FE6B54CD460
[07/16/2025 03:09:50:290 PDT] 037 StreamMonitor                  process-output                 I RIP=00007FE6B87F23C0 GS=0000 FS=0000 RSP=00007FE6B8C59998

...
[07/16/2025 03:09:50:307 PDT] 037 StreamMonitor                  process-output                 I JVMDUMP039I Processing dump event "gpf", detail "" at 2025/07/16 03:09:50 - please wait.
[07/16/2025 03:09:50:315 PDT] 037 StreamMonitor                  process-output                 I JVMDUMP032I JVM requested System dump using '/home/ci/Build/workspace/ebcTestRunner/dev/autoFVT/io.openliberty.microprofile.rest.client.3.0.internal_fat_tck/autoFVT/publish/tckRunner/tck/core.20250716.030950.18025.0001.dmp' in response to an event
[07/16/2025 03:09:54:275 PDT] 037 StreamMonitor                  process-output                 I JVMPORT030W /proc/sys/kernel/core_pattern setting "|/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.20189.  Review the manual for the external program to find where the core dump is written and ensure the program does not truncate it.
[07/16/2025 03:10:00:086 PDT] 037 StreamMonitor                  process-output                 I JVMPORT049I The core file created by child process with pid = 20189 was not found. Review the documentation for the /proc/sys/kernel/core_pattern program "|/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E" to find where the core file is written and ensure that program does not truncate it.
[07/16/2025 03:10:00:086 PDT] 037 StreamMonitor                  process-output                 I JVMDUMP012E Error in System dump: /home/ci/Build/workspace/ebcTestRunner/dev/autoFVT/io.openliberty.microprofile.rest.client.3.0.internal_fat_tck/autoFVT/publish/tckRunner/tck/core.20250716.030950.18025.0001.dmp
[07/16/2025 03:10:00:086 PDT] 037 StreamMonitor                  process-output                 I JVMDUMP032I JVM requested Java dump using '/home/ci/Build/workspace/ebcTestRunner/dev/autoFVT/io.openliberty.microprofile.rest.client.3.0.internal_fat_tck/autoFVT/publish/tckRunner/tck/javacore.20250716.030950.18025.0002.txt' in response to an event

...
```

--- a/dev/com.ibm.ws.microprofile.rest.client13_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
+++ b/dev/com.ibm.ws.microprofile.rest.client13_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
@@ -16,6 +16,11 @@
         <packages>
             <package name="org.eclipse.microprofile.rest.client.tck.*"></package>
         </packages>
+        <!-- ******Begin of Fips-related excludes****** 
+             The keystore using old PBE algorithm is no longer supported with FIPS 140-3, 
+             the related tests from *.tck.ssl.* are temporarily excluded until the issue is fixed, 
+             which is currently tracked under https://github.com/microprofile/microprofile-rest-client/issues/399. 
+        -->
         <classes>
           <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslContextTest">
             <methods>
@@ -38,5 +43,6 @@
             </methods>
           </class>
         </classes>
+        <!-- ******End of Fips-related excludes****** -->
     </test>
 </suite>

--- a/dev/com.ibm.ws.microprofile.rest.client14_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
+++ b/dev/com.ibm.ws.microprofile.rest.client14_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
@@ -16,6 +16,11 @@
         <packages>
             <package name="org.eclipse.microprofile.rest.client.tck.*"></package>
         </packages>
+        <!-- ******Begin of Fips-related excludes****** 
+             The keystore using old PBE algorithm is no longer supported with FIPS 140-3, 
+             the related tests from *.tck.ssl.* are temporarily excluded until the issue is fixed, 
+             which is currently tracked under https://github.com/microprofile/microprofile-rest-client/issues/399. 
+        -->
         <classes>
           <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslContextTest">
             <methods>
@@ -38,5 +43,6 @@
             </methods>
           </class>
         </classes>
+        <!-- ******End of Fips-related excludes****** -->
     </test>
 </suite>

--- a/dev/io.openliberty.microprofile.rest.client.2.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
+++ b/dev/io.openliberty.microprofile.rest.client.2.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
@@ -17,36 +17,42 @@
             <package name="org.eclipse.microprofile.rest.client.tck.*"/>
         </packages>
         <classes>
-        <class name="org.eclipse.microprofile.rest.client.tck.sse.BasicReactiveStreamsTest">
-            <methods>
+          <class name="org.eclipse.microprofile.rest.client.tck.sse.BasicReactiveStreamsTest">
+             <methods>
                 <exclude name=".*" />
-            </methods>
-        </class>
-        <class name="org.eclipse.microprofile.rest.client.tck.sse.ReactiveStreamsPublisherTckTest">
-            <methods>
+             </methods>
+          </class>
+          <class name="org.eclipse.microprofile.rest.client.tck.sse.ReactiveStreamsPublisherTckTest">
+             <methods>
                 <exclude name=".*" />
-            </methods>
-        </class>
-        <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslContextTest">
-            <methods>
+             </methods>
+          </class>
+          <!-- ******Begin of Fips-related excludes****** 
+               The keystore using old PBE algorithm is no longer supported with FIPS 140-3, 
+               the related tests from *.tck.ssl.* are temporarily excluded until the issue is fixed, 
+               which is currently tracked under https://github.com/microprofile/microprofile-rest-client/issues/399. 
+          -->
+          <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslContextTest">
+             <methods>
                 <exclude name="shouldSucceedMutualSslWithValidSslContext" />
-            </methods>
+             </methods>
           </class>
           <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslHostnameVerifierTest">
-            <methods>
+             <methods>
                 <exclude name=".*" />
-            </methods>
+             </methods>
           </class>
           <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslMutualTest">
-            <methods>
+             <methods>
                 <exclude name=".*" />
-            </methods>
+             </methods>
           </class>
           <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslTrustStoreTest">
-            <methods>
+             <methods>
                 <exclude name=".*" />
-            </methods>
+             </methods>
           </class>
-    </classes>
+          <!-- ******End of Fips-related excludes****** -->
+        </classes>
     </test>
 </suite>

--- a/dev/io.openliberty.microprofile.rest.client.3.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
+++ b/dev/io.openliberty.microprofile.rest.client.3.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
@@ -17,17 +17,22 @@
             <package name="org.eclipse.microprofile.rest.client.tck.*"/>
         </packages>
         <classes>
-        <class name="org.eclipse.microprofile.rest.client.tck.sse.BasicReactiveStreamsTest">
+          <class name="org.eclipse.microprofile.rest.client.tck.sse.BasicReactiveStreamsTest">
             <methods>
                 <exclude name=".*" />
             </methods>
-        </class>
-        <class name="org.eclipse.microprofile.rest.client.tck.sse.ReactiveStreamsPublisherTckTest">
+          </class>
+          <class name="org.eclipse.microprofile.rest.client.tck.sse.ReactiveStreamsPublisherTckTest">
             <methods>
                 <exclude name=".*" />
             </methods>
-        </class>
-        <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslContextTest">
+          </class>
+          <!-- ******Begin of Fips-related excludes****** 
+               The keystore using old PBE algorithm is no longer supported with FIPS 140-3, 
+               the related tests from *.tck.ssl.* are temporarily excluded until the issue is fixed, 
+               which is currently tracked under https://github.com/microprofile/microprofile-rest-client/issues/399.
+          -->
+          <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslContextTest">
             <methods>
                 <exclude name="shouldSucceedMutualSslWithValidSslContext" />
             </methods>
@@ -47,6 +52,7 @@
                 <exclude name=".*" />
             </methods>
           </class>
+          <!-- ******End of Fips-related excludes****** -->
         </classes>
     </test>
 </suite>

--- a/dev/io.openliberty.microprofile.rest.client.4.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
+++ b/dev/io.openliberty.microprofile.rest.client.4.0.internal_fat_tck/publish/tckRunner/tck/tck-suite.xml-fips
@@ -1,4 +1,4 @@
-<!--Copyright (c) 2024, 2205 IBM Corporation and others. All rights reserved.
+<!--Copyright (c) 2024, 2025 IBM Corporation and others. All rights reserved.
     This program and the accompanying materials are made available under the 
     terms of the Eclipse Public License 2.0 which accompanies this distribution, 
     and is available at 
@@ -16,6 +16,11 @@
         <packages>
             <package name="org.eclipse.microprofile.rest.client.tck.*"/>
         </packages>
+        <!-- ******Begin of Fips-related excludes****** 
+             The keystore using old PBE algorithm is no longer supported with FIPS 140-3,
+             the related tests from *.tck.ssl.* are temporarily excluded until the issue is fixed,
+             which is currently tracked under https://github.com/microprofile/microprofile-rest-client/issues/399.
+        -->
         <classes>
           <class name="org.eclipse.microprofile.rest.client.tck.ssl.SslContextTest">
             <methods>
@@ -38,5 +43,6 @@
             </methods>
           </class>
         </classes>
+        <!-- ******End of Fips-related excludes****** -->
     </test>
 </suite>

